docs/searchengine_deployemnt.md

```diff
@@ -0,0 +1,21 @@
+Searchengine installation and configuration using Ansible:
+==========================================================
+
+There is an ansible playbook (management-searchengine.yml) that has been written to deploy the apps:
+* It will config and run searchengine, Elasticsearch and searchengine client
+* It will configure and create the required folders
+* It will configure the three apps and run them
+* There is a variables file (searchengine_vars.yml) that the user needs to edit before running the playbook
```

Comment:
This file should be renamed as `groups_vars/management-hosts.yml` until we decide on a better group/infrastructure